### ISSUE TITLE
[Bug] 模型微调后进行推理，返回全是感叹号！

### ISSUE THREAD

**Issue by ChenJian7578** [OpenGVLab/InternVL#733]
### Checklist

- [X] 1. I have searched related issues but cannot get the expected help.
- [X] 2. The bug has not been fixed in the latest version.
- [X] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.

### Describe the bug

请问我在用CLoT_cn_2000数据集微调后进行推理时，返回全是感叹号是怎么回事？又遇到这个情况吗，使用的是lora微调1B的模型

### Reproduction

！

### Environment

```Shell
！
```

**Comment by czczup** [comment]
您好，感谢反馈，请发一下您使用的shell脚本吗

**Comment by awakenlee180** [comment]
我这边也是

**Comment by awakenlee180** [comment]
我的shell脚本：
set -x

GPUS=${GPUS:-2}
BATCH_SIZE=${BATCH_SIZE:-2}
PER_DEVICE_BATCH_SIZE=${PER_DEVICE_BATCH_SIZE:-1}
GRADIENT_ACC=$((BATCH_SIZE / PER_DEVICE_BATCH_SIZE / GPUS))


export PYTHONPATH="${PYTHONPATH}:$(pwd)"
export MASTER_PORT=34228
export TF_CPP_MIN_LOG_LEVEL=3
export LAUNCHER=pytorch

OUTPUT_DIR='work_dirs/internvl_chat_v2_0/internvl2_1b_qwen2_0_5b_dynamic_res_2nd_finetune_lora'

if [ ! -d "$OUTPUT_DIR" ]; then
  mkdir -p "$OUTPUT_DIR"
fi

# number of gpus: 2
# batch size per gpu: 4
# gradient accumulation steps: 2
# total batch size: 16
# epoch: 1
torchrun \
  --nnodes=1 \
  --node_rank=0 \
  --master_addr=127.0.0.1 \
  --nproc_per_node=${GPUS} \
  --master_port=${MASTER_PORT} \
  internvl/train/internvl_chat_finetune.py \
  --model_name_or_path "/DATA/workshop/personal/InternVL-main/pretrained/InternVL2-1B" \
  --conv_style "Hermes-2" \
  --output_dir ${OUTPUT_DIR} \
  --meta_path "/DATA/jupyter/personal/InternVL-main/internvl_chat/shell/data/mydata.json" \
  --overwrite_output_dir True \
  --force_image_size 448 \
  --max_dynamic_patch 1 \
  --down_sample_ratio 0.5 \
  --drop_path_rate 0.0 \
  --freeze_llm True \
  --freeze_mlp True \
  --freeze_backbone True \
  --use_llm_lora 16 \
  --vision_select_layer -1 \
  --dataloader_num_workers 1 \
  --bf16 False \
  --num_train_epochs 1 \
  --per_device_train_batch_size ${PER_DEVICE_BATCH_SIZE} \
  --gradient_accumulation_steps ${GRADIENT_ACC} \
  --evaluation_strategy "no" \
  --save_strategy "steps" \
  --save_steps 200 \
  --save_total_limit 1 \
  --learning_rate 4e-5 \
  --weight_decay 0.01 \
  --warmup_ratio 0.03 \
  --lr_scheduler_type "cosine" \
  --logging_steps 1 \
  --max_seq_length 4096 \
  --do_train True \
  --grad_checkpoint True \
  --group_by_length True \
  --dynamic_image_size True \
  --use_thumbnail True \
  --ps_version 'v2' \
  --deepspeed "zero_stage1_config.json" \
  --report_to "tensorboard" \
  2>&1 | tee -a "${OUTPUT_DIR}/training_log.txt"

**Comment by awakenlee180** [comment]
我推理的时候路径就是写的这个work_dirs/internvl_chat_v2_0/internvl2_1b_qwen2_0_5b_dynamic_res_2nd_finetune_lora,然后用quickstart里面的推理脚本，会给我报错 有几个py文件没有 我就把pretrained里面的对应文件复制过去然后运行，返回的都是感叹号，是有什么没有配置嘛？我之前在3群里问过 没给我回复

**Comment by ChenJian7578** [comment]
因为你没有合并lora训练的权重，参考：https://internvl.readthedocs.io/en/latest/tutorials/coco_caption_finetune.html
![image](https://github.com/user-attachments/assets/39bf6de9-ff2e-4c76-918c-b9972ead58de)
